sdk/appconfiguration/app-configuration/test/index.spec.ts

```diff
@@ -781,45 +783,7 @@ describe("AppConfigurationClient", () => {
         result.contentType,
```

Comment:
I can't leave a comment on the exact line, but you should be able to remove the entire `updateConfigurationSetting` `describe` section since you've removed that method altogether.

Reply:
You are 100% right - GONE.